testprojects/src/python/BUILD

```diff
@@ -21,6 +21,6 @@ files(name="sources_directory", sources=["sources/**/*"])
 
 experimental_run_shell_command(
     name="cmd",
-    command='{{ tree {chroot}; echo "in: $CHROOT"; }} | tee output.log',
+    command='{ find {chroot} -type f; echo "in: $CHROOT"; } | tee output.log',
```

Comment:
If this is not going through formatting, then the `{chroot}` part is hardly correct either.

Reply:
All I know is, it works (on macos and linux) without the double braces and does not work with them.

Reply:
Looks like we use string replace to replace `{chroot}` with the chroot path, not a call to `format()`:

https://github.com/pantsbuild/pants/blob/756a4489bba88690fda4b6c2366e19acde698e2f/src/rust/engine/process_execution/src/local.rs#L590

Reply:
Oohh... that would explain it.